src/main/resources/hackathon-data/courts2.csv

```diff
@@ -0,0 +1,82 @@
+id,latitude,longitude,address,city,state,zip_code

```

Comment:
All the CSV files should have been removed in favor of just SQL Insert Statements.....